### PR TITLE
Fixed for new database structure

### PR DESCRIPTION
**Pull request by MrGorbunov**
For the 2023 FIA database, the FVS_STANDINIT_PLOT STAND_IDs 
must be unique (which no other table does ?) and this would mess
up the script when changing STAND_IDs to FOR_TYPEs.

Now, a block of SQL gets run which removes the UNIQUE constraint
from FVS_STANDINIT_PLOT.STAND_ID.

### CODE REVIEW

**Comment by MrGorbunov** [comment]
Actually hold off on merging for a bit, on Monday or Tuesday I'm going to test this with Lauren and once it works for her we should merge.

**Comment by MrGorbunov** [comment]
Yes it is good to merge

**Review by wzip — APPROVED** [review]

**Pull request merged**
